### PR TITLE
feat(Messages): Adds copy text option to message context menu

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖
image

### CODE REVIEW

File: ui/src/layouts/chats/presentation/messages/mod.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@ use std::{
 };
 
 use dioxus::prelude::{EventHandler, *};
+use arboard::Clipboard;
```
Comment (stavares843) [inline ui/src/layouts/chats/presentation/messages/mod.rs:7]:
```suggestion
use dioxus::prelude::{EventHandler, *};
```


to fix the linter issue 🔨 

image

**Review by sdwoodbury — CHANGES_REQUESTED** [review]
request you don't use `unwrap()` here and also fix the formatting or clippy, so that the clippy workflow passes.

**Comment by InfamousVague** [comment]
> request you don't use `unwrap()` here and also fix the formatting or clippy, so that the clippy workflow passes.

How do we wanna handle this? We're currently unwrapping every use case of arboard can go ahead and implement a sweeping change

**Review by sdwoodbury — APPROVED** [review]

File: ui/src/components/friends/add.rs [hunk 2]
```diff
@@ -72,8 +72,16 @@ pub fn AddFriend(cx: Scope) -> Element {
...
 
     if let Some(id) = my_id.get().clone() {
-        let mut clipboard = Clipboard::new().unwrap();
-        clipboard.set_text(id).unwrap();
+        match Clipboard::new() {
+            Ok(mut c) => {
+                if let Err(e) = c.set_text(id) {
+                    log::warn!("Unable to set text to clipboard: {e}");
+                }
+            },
```
Comment (stavares843) [inline ui/src/components/friends/add.rs]:
```suggestion
            }
```

to fix the linter issue 🔨 

Captura de ecrã 2023-11-09, às 22 16 19

**Review by stavares843 — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Pull request merged**